app/graphql/context.js

```diff
@@ -1,11 +1,17 @@
 import { RuralPaymentsAgencyLandAPI } from '../data-sources/rural-payments-agency-land-api.js'
 import { RuralPaymentsPortalApi } from '../data-sources/rural-payments-portal/RuralPaymentsPortalApi.js'
+import { CustomerApplicationsDataSource } from './data-source/customer'
 
 export function context () {
   return {
     dataSources: {
       ruralPaymentsAgencyLandAPI: new RuralPaymentsAgencyLandAPI(),
-      ruralPaymentsPortalApi: new RuralPaymentsPortalApi()
+      ruralPaymentsPortalApi: new RuralPaymentsPortalApi(),
+      rest: {
```

Comment:
I don't think custom applications is a separate data source from  RuralPaymentsPortalApi?
If we do need this we should either restructure the other code or follow the pattern laid out by others